invenio_cli/helpers/docker_helper.py

```diff
@@ -43,6 +43,20 @@ def _normalize_name(self, project_shortname):
         else:
             return project_shortname
 
+    def _get_container_from_service(self, service_name):
+        """Retrieve the docker container for the given service_name."""
+        container = None
```

Comment:
minor: There is an outer scope `container` variable, which is only used for a return at the end. However, there is anotther `container` variable inside the for loop... Could we avoid the outer `container-=None` and just have at the end:

```python
if container_name:
    return self.docker_client...
    
return None
```

Or a one liner `return None if not container_name else self.docker_client...`